src/Rules/PHPUnit/DataProviderHelper.php

```diff
@@ -5,16 +5,32 @@
 use PHPStan\Analyser\Scope;
 use PHPStan\PhpDoc\ResolvedPhpDocBlock;
 use PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTagNode;
+use PHPStan\Reflection\ClassReflection;
 use PHPStan\Reflection\MissingMethodFromReflectionException;
+use PHPStan\Reflection\ReflectionProvider;
 use PHPStan\Rules\RuleError;
 use PHPStan\Rules\RuleErrorBuilder;
 use function array_merge;
+use function count;
+use function explode;
 use function preg_match;
 use function sprintf;
 
 class DataProviderHelper
 {
 
+	/**
+	 * Reflection provider.
+	 *
+	 * @var ReflectionProvider
+	 */
+	private $reflectionProvider;
+
+	public function __construct(ReflectionProvider $reflectionProvider)
+	{
+		$this->reflectionProvider = $reflectionProvider;
```

Comment:
So there is auto wiring it the project? I wanted to fix it by myself, but failed on getting reflection of other class and it is that simple :)

Reply:
https://phpstan.org/developing-extensions/dependency-injection-configuration#services